meeteval/wer/wer/mimo.py

```diff
@@ -90,11 +50,11 @@ def get_keys(obj):
     )
 
     er = sum([
-        siso_word_error_rate(
-            ' '.join(r),  # Concatenate all utterances from one speaker
+        _siso_error_rate(
+            [t for r_ in r for t in r_],  # Concatenate all utterances from one speaker
```

Comment:
`# Concatenate all utterances from one speaker` -> `# Create list with all words from one speaker`?

Reply:
It's not necessarily words at this point. The function accepts anything, so it could be characters here

Reply:
Yes, I agree, that it could be anything that is hashable. But I didn't know, how to phrase it, so I used the most common case and that is: word.